SWIPs/swip-0.md

```diff
@@ -8,9 +8,9 @@ created: 2019-07-08
 updated: 2019-07-09
```

Comment:
This seems like it should be in another PR, can we move it please?

Reply:
+1